src/app/cli/src/coda_main.ml

```diff
@@ -734,8 +734,13 @@ struct
         ~frontier_broadcast_pipe =
       match%map Reader.load_bin_prot disk_location Pool.bin_reader_t with
       | Ok pool ->
-          of_pool_and_diffs pool ~parent_log ~incoming_diffs
-            ~frontier_broadcast_pipe
+          let network_pool =
+            of_pool_and_diffs pool ~parent_log ~incoming_diffs
+          in
+          ignore
```

Comment:
I thought this already returns unit, it probably can if it doesn't

Reply:
it does, I removed the `ignore`